sdk/identity/identity/test/internal/node/managedIdentityCredential.spec.ts

```diff
@@ -246,6 +244,49 @@ describe("ManagedIdentityCredential", function() {
           `Failed to retrieve IMDS token after ${imdsMsiRetryConfig.maxRetries} retries.`
         ) > -1
     );
+    clock?.restore();
+  });
+
+  it("IMDS MSI retries also retries on 503s", async function() {
+    process.env.AZURE_CLIENT_ID = "errclient";
```

Comment:
Here we update process.env. Below we pass `process.env.AZURE_CLIENT_ID` to ManagedIdentityCredential constructor.  Is the environment variable still retrieved by some code?

Reply:
Thank you! 🙏